modules/ROOT/pages/faq.adoc

```diff
@@ -123,9 +123,10 @@ write and modify files.
 
 The directory `/etc` may be changed by deployments, but will not override user
 made changes. The content under `/var` is left untouched by rpm-ostree when
-applying upgrades or rollbacks. For more information, refer to the
-https://rpm-ostree.readthedocs.io/en/latest/manual/administrator-handbook/#filesystem-layout[filesystem
-layout].
+applying upgrades or rollbacks. For more information, refer to the 
+https://docs.fedoraproject.org/en-US/fedora-coreos/storage/#_mounted_filesystems[Filesystem configuration]
```

Comment:
WDYT about

```suggestion
https://docs.fedoraproject.org/en-US/fedora-coreos/storage/#_mounted_filesystems[Mounted Filesystems]
```

instead to match the title of the linked section?

Reply:
I think *« Filesystem configuration »* is more understandable than *« Mounted Filesystems »*. In my opinion, the linked chapter is more about *« Filesystem configuration »* than *« Mounted Filesystems »*.

However you can change my edit as you wish, I do not care. :wink:

Reply:
I think the link name should match the name of the section to which it's linking. You're right that section could probably be renamed "Filesystem Configuration" instead.

Reply:
Want to push another commit to rename it in this PR? If not, I can pick that up.

Reply:
Yes please pick up this PR. :+1:

Reply:
OK, tweaked this to just drop the upstream rpm-ostree link, and renamed that section. (I think that text dates back from before we had that section written up).